src/libs/kata-types/src/capablities.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use bitmask_enum::bitmask;
+
+#[bitmask(u8)]
+enum CapabilityBits {
+    blockDeviceSupport,
+    blockDeviceHotplugSupport,
+    multiQueueSupport,
+    fsSharingSupported,
```

Comment:
- Others are using `Support` suffix, but `fsSharing` is using `Supported`.
- Should they use an uppercase capital? For example `BlockDeviceSupport`

Reply:
fixed